### ISSUE TITLE
Error with test-kitchen 1.17.0

### ISSUE THREAD

**Issue by jesusvazquez** [neillturner/kitchen-ansible#255]
Hello!

Today we got a couple of errors in our travis ci related with kitchen-ansible.

ATM we're installing the latest package of chefdk `2.1.11-1` which installs the version `1.17.0` of test-kitchen.

Current versions:
```
kitchen_version 1.17.0
kitchen-ansible 0.47.3
```

When running `kitchen diagnose --all` we get the following error: 
``` 
kitchen_version: 1.17.0
plugins:
  error:
    exception: "#<Kitchen::ClientError: Could not load the 'ansible_playbook' provisioner
      from the load path. Please ensure that your provisioner is installed as a gem
      or included in your Gemfile if using Bundler.>"
    message: Could not load the 'ansible_playbook' provisioner from the load path.
      Please ensure that your provisioner is installed as a gem or included in your
      Gemfile if using Bundler.
    backtrace:
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/provisioner.rb:46:in
      `rescue in for_plugin'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/provisioner.rb:38:in
      `for_plugin'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/config.rb:283:in
      `new_provisioner'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/config.rb:245:in
      `new_instance'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/config.rb:137:in
      `block in build_instances'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/config.rb:136:in
      `map'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/config.rb:136:in
      `with_index'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/config.rb:136:in
      `build_instances'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/config.rb:112:in
      `instances'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:112:in
      `filtered_instances'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:142:in
      `parse_subcommand'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command/diagnose.rb:53:in
      `load_instances'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command/diagnose.rb:32:in
      `block in call'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command/diagnose.rb:73:in
      `record_failure'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command/diagnose.rb:32:in
      `call'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/cli.rb:53:in
      `perform'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/cli.rb:143:in
      `diagnose'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/command.rb:27:in
      `run'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in
      `invoke_command'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/cli.rb:334:in
      `invoke_task'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor.rb:359:in
      `dispatch'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/base.rb:440:in
      `start'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/bin/kitchen:13:in
      `block in <top (required)>'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/errors.rb:171:in
      `with_friendly_errors'"
    - "/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/bin/kitchen:13:in
      `<top (required)>'"
    - "/opt/chefdk/bin/kitchen:263:in `load'"
    - "/opt/chefdk/bin/kitchen:263:in `<main>'"
``` 

We know that 1.17 was released 3 days ago and thats why its probably broken but I just wanted to report the error. 

### Workaround

Fix chefdk version to `2.0.28-1` which will install test-kitchen version `1.16.0` and will work just fine.

**Comment by neillturner** [comment]
thanks for that. i will test with test kitchen  1.17

**Comment by neillturner** [comment]
closed. fixed in version 0.47.4
